@AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mpd_defines.hpp

```diff
@@ -353,7 +360,7 @@
 #define MPD_ARROW_R(name, startX, startY, numChars) class Mpd_Arrow_##name##_Line { \
     type = line; \
     points[] = { \
-        {{startX, startY - MPD_ARROW_PAD}, 1}, \
+        {{startX + MPD_ARROW_LENGTH, startY - MPD_ARROW_PAD}, 1}, \
```

Comment:
Why was this changed?

Reply:
This was meant to fix an issue where the arrow wasn't starting at the exact left edge IIRC.

Reply:
This seems to move the left point too far over on my computer? The tip of the arrow is in-line, but the left is pushed in

![20230205003327_1](https://user-images.githubusercontent.com/9770674/216795461-6b3360cf-61df-47a4-8015-403d54d508a6.jpg)

Reply:
Reverted, seems to be working as intended.